src/ui/components/RightSidebar/Comment.js

```diff
@@ -81,16 +81,15 @@ class Comment extends React.Component {
 
   renderAvatar() {
     const { comment } = this.props;
-
-    // Comments that have been made prior to adding the users feature don't
-    // have an associated user. We just give those comments a grey circle.
-    // This should be removed eventually.
+    const { picture, name } = comment.user;
```

Comment:
Is it possible to have a comment without an avatar?

```
comment?.user.avatarID`
```

makes it look like it is, but who knows

Reply:
In a way, yes. 

Old comments from before we enabled the users feature don't have a user property. Because of that, there's no corresponding avatarID for that comment. We use the avatarID to generate the avatar color, so in a way the old comments don't have an avatar.

I'll rewrite this as `comment.user?.avatarID` since `comment.user` can be undefined.